ejemplo.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>No lo se aun</title>
+	<link rel="stylesheet" href="styles.css" />
+</head>
+<body>
+	<section>
+		<figure class="ejemplo">
+			<img src="phone.png" width="foto" height="foto" alt="foto"  />
```

Comment:
width="foto", height="foto" ?? whats that?. Style things like height or width should be defined through css, html is just for structuring stuff.